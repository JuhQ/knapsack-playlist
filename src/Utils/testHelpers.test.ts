import generateTestData from "./testHelpers";

describe("Test helpers", () => {
  it("should generate test data", () => {
    expect(generateTestData(20, 20).all()).toMatchSnapshot();
  });
});
