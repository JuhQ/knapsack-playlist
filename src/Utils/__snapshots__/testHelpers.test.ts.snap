// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test helpers should generate test data 1`] = `
Array [
  Object {
    "id": "0",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "1",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "2",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "3",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "4",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "5",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "6",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "7",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "8",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "9",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "10",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "11",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "12",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "13",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "14",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "15",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "16",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "17",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "18",
    "seconds": 20,
    "title": "test",
  },
  Object {
    "id": "19",
    "seconds": 20,
    "title": "test",
  },
]
`;
